scripts/mdbook-forc-documenter/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "mdbook-forc-documenter"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[[bin]]
+name = "mdbook-forc-documenter"
+path = "src/bin/mdbook-forc-documenter.rs"
+
+[lib]
+name = "mdbook_forc_documenter"
+path = "src/lib.rs"
+
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3.1", features = [ "derive" ]  }
+mdbook = "=0.4.15"
```

Comment:
Do we need to pin to `=0.4.15` in particular? Or could we specify `0.4.15` to allow for using that version and above?

Reply:
Also are we able to disable default features here? E.g.

```toml
mdbook = { version = "0.4.15", default-features = false }
```

My guess is we don't need [these features](https://github.com/rust-lang/mdBook/blob/bf258eeb9b245b9c329e147dd1884307c5ef7644/Cargo.toml#L60) to build our forc documenter preprocessor, as the actual `mdbook` tool will have those features anyway?

Hopefully disabling default features might help to cut down on compilation time a bit by removing the need for those extra crates. That said, they'll probably still show up in our `Cargo.lock` either way.

Reply:
> Do we need to pin to `=0.4.15` in particular? Or could we specify `0.4.15` to allow for using that version and above?

Hmm nope, realized that was a mistake on my part because I had the wrong version installed and it was throwing a warning, just had to update it

> My guess is we don't need [these features](https://github.com/rust-lang/mdBook/blob/bf258eeb9b245b9c329e147dd1884307c5ef7644/Cargo.toml#L60) to build our forc documenter preprocessor, as the actual mdbook tool will have those features anyway?

Thanks for this good point! Yup, seems like we only require the `build` feature, so we can go ahead and remove the features.

Reply:
Great news, looks like that did slim down `Cargo.lock` quite a bit too!